src/Symfony/Bridge/Doctrine/Tests/PropertyInfo/Fixtures/Foo.php

```diff
@@ -16,8 +16,15 @@
  */
 class Foo
 {
+    const TYPE_A = 'a';
+
     /**
      * @var string
      */
     public $bar;
+
+    /**
+     * @var string
+     */
+    public $baz;
```

Comment:
why did you change this file?

Reply:
Yep that doesn't look right. I don't remember, could be oversight indeed. I've removed it, I hope tests still pass